modules/indigo/module/inc/indigo/of_connection_manager.h

```diff
@@ -814,6 +814,49 @@ indigo_cxn_subbundle_set2(uint32_t num_subbundles,
                           indigo_cxn_subbundle_start_t starts[],
                           indigo_cxn_subbundle_finish_t finishes[]);
 
+
+/**
+ * subbundle functions are used to communicate with driver that the
+ * bundle task is going to start the subbundle process and bundle task
+ * has processed the last message in the subbundle.
+ * Driver can take the proper action accordingly.
+ */
+typedef struct indigo_cxn_subbundle_info_s {
+    uint32_t subbundle_idx;
+    uint32_t total_msg_count;
+    uint32_t cur_msg_count;
+} indigo_cxn_subbundle_info_t;
+
+typedef void (*indigo_cxn_subbundle_pre_start_t)(indigo_cxn_id_t cxn_id,
+                                                 indigo_cxn_subbundle_info_t *subbundle_info);
+typedef void (*indigo_cxn_subbundle_post_finish_t)(indigo_cxn_id_t cxn_id,
+                                                   indigo_cxn_subbundle_info_t *subbundle_info);
+
+/**
+ * Extends the subbundle infrastructure from indigo_cxn_subbundle_set2 above.
+ * First five parameters are the same as for indigo_cxn_subbundle_set2.
+ * @param pre_starts Array of prestart functions; the ith function is called
```

Comment:
let's clean up all instances of the prestart/postfinish language, which will just serve to confuse

Reply:
not sure why you still see the stale codes. They have been removed